### PR TITLE
update example URL for redfish driver

### PR DESCRIPTION
**Pull request by dhellmann**
Use the same path that is found in the Ironic documentation at
https://docs.openstack.org/ironic/latest/admin/drivers/redfish.html

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ mainly, but not only, provisioning details.
     * Redfish
       * `redfish://` (or `redfish+http://` to disable TLS), the hostname
         or IP address, and the path to the system ID are required,
-        for example `redfish://myhost.example/redfish/v1/Systems/MySystemExample`
+        for example `redfish://myhost.example/redfish/v1/Systems/1`
```
Comment (stbenjam) [inline docs/api.md]: The real world example we have is `/redfish/v1/Systems/System.Embedded.1`, would that be better?

Comment (stbenjam) [reply docs/api.md]: I was just thinking we might want to indicate it doesn't need to be numeric

However, https://docs.openstack.org/ironic/latest/admin/drivers/redfish.html shows your example here, so maybe it's a better idea to match the Ironic docs.

Comment (dhellmann) [reply docs/api.md]: I've updated the patch to replace the existing value with a real one and to add the example from the ironic documentation.

Comment (stbenjam) [reply docs/api.md]: :+1:

**Pull request merged**